tommoor/fontselect-jquery-plugin
# Automatically close dropdown on document click

This branch contains the following functionality:
* When a user clicks on the document away from the dropdown, the dropdown will automatically close
* If multiple dropdown are present, selecting one will close any that are already open
* Resolves Issue #7 

## How to test
* Dropdown should still maintain original functionality for font selection
* When the dropdown is focused, if you click any other elements on the page, it should automatically close

## Screenshots / Recordings
![pechwvn8ku](https://user-images.githubusercontent.com/16951595/31305924-77c2222a-aaf9-11e7-8821-14e73176d0e0.gif)
